fulp_modules/main_features/bloodsuckers/code/antagonists/bloodsucker_datum.dm

```diff
@@ -769,6 +769,61 @@
 
 	return fullname
 
+/// Special midround/latejoin only bloodsucker - used to override procs accordingly.
+/datum/antagonist/bloodsucker/tzimisce
```

Comment:
I dont like this at all. you can do this instead

/datum/dynamic_ruleset/midround/tzimisce_bloodsucker/execute()
	var/mob/M = pick(living_players)
	assigned += M.mind
	living_players -= M.mind
	var/datum/antagonist/bloodsucker/sucker = new
	M.mind.add_antag_datum(sucker)
	sucker.my_clan = CLAN_TZIMISCE
	message_admins("[ADMIN_LOOKUPFLW(M)] was selected by the [name] ruleset and has been made into a midround Bloodsucker.")
	log_game("DYNAMIC: [key_name(M)] was selected by the [name] ruleset and has been made into a midround Bloodsucker.")
	return TRUE

From there, we can change AssignClanAndBane accordingly to fit with these changes, like not assigning a clan if you're already in one.